apps/govquests-api/rails_app/db/seeds.rb

```diff
@@ -107,3 +107,11 @@ def associate_action_with_quest(quest_id, action_id, position)
 end
 
 puts "Quests created and actions associated successfully!"
+
+Rails.configuration.command_bus.call(Authentication::RegisterUser.new(
+  user_id: SecureRandom.uuid,
+  email: "jose@bleu.studio",
+  user_type: "non_delegate",
+  wallet_address: "0x26394373F96950025DA55b07809c976a4768c995",
```

Comment:
use whoever is developing wallet to allow them to test the gitcoin functionality. In the future, remove this and only rely on users registered on the platform